compiler/code-gen/declarations.cpp

```diff
@@ -1141,16 +1141,26 @@ void ClassMembersDefinition::compile_msgpack_serialize(CodeGenerator &W, ClassPt
   //   packer.pack(field_1);
   //   ...
   //}
-  std::vector<std::string> body;
+  std::list<std::string> body;
```

Comment:
Why not a `std::deque`?

Reply:
Deque is excessive here. Moreover, I believe `std::forward_list` is cheaper, one might try.

Reply:
Almost the only case where `std::list` is preferable over `std::vector` or `std::deque` is when you need to insert/remove elements in middle positions.